tests/tests_pytorch/callbacks/test_stochastic_weight_avg.py

```diff
@@ -255,3 +281,103 @@ def on_train_epoch_start(self):
     )
     trainer.fit(model)
     assert model.on_train_epoch_start_called
+
+
+def _swa_resume_training_from_checkpoint(tmpdir, model, resume_model, ddp=False):
+    swa_start = 3
+    trainer_kwargs = {
+        "default_root_dir": tmpdir,
+        "max_epochs": 5,
+        "accelerator": "cpu",
+        "strategy": "ddp_spawn_find_unused_parameters_false" if ddp else None,
+        "devices": 2 if ddp else 1,
+        "limit_train_batches": 5,
+        "limit_val_batches": 0,
+        "accumulate_grad_batches": 2,
+        "enable_progress_bar": False,
+    }
+    trainer = Trainer(callbacks=SwaTestCallback(swa_epoch_start=swa_start, swa_lrs=0.1), **trainer_kwargs)
+
+    with _backward_patch(trainer), pytest.raises(Exception if ddp else DummyError):
```

Comment:
you can raise Exception(msg) instead and match it with the error msg
```python
pytest.raises(..., match='Crash test')
```

Reply:
Yeah that's probably better. I was previously raising a `RuntimeError` but Carlos asked to have this changed to a custom dummy exception class to make sure we can only catch the specific exception we're expecting. After adding tests with DDP I ran into the problem that the exceptions that are thrown in subprocesses get wrapped in a plain `Exception` object though, so the type is different. Just always raising an `Exception` with a message to match on seems reasonable to simplify this.